nsqadmin/static/js/views/channel.hbs

```diff
@@ -178,7 +178,7 @@
             </tr>
             {{#each clients}}
             <tr>
-                <td title="{{remote_address}}">{{client_id}}</td>
+                <td title="{{remote_address}}">{{hostname_port}}{{#if client_id.length}} ({{client_id}}){{/if}}</td>
```

Comment:
I believe `{{#if client_id}}` should be sufficient if you're just trying to catch empty strings. Either way is fine though.